airflow/bin/cli.py

```diff
@@ -607,6 +615,11 @@ class CLIFactory(object):
             ("--stdout", ), "Redirect stdout to this file"),
         'log_file': Arg(
             ("-l", "--log-file"), "Location of the log file"),
+        'yes': Arg(
+            ("-y", "--yes"),
+            "Do not prompt to confirm. Use with care!",
+            "store_true",
```

Comment:
it means that you do not need to specify a value after the flag, the presence of the CLI flag will set the associated value to `True`